Week 3/Magic8Ball.html

```diff
@@ -0,0 +1,63 @@
+<html>
+   <head>
+         <script
+    src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js">
+    </script>
+   </head>
+   <body>
+   	<link rel="stylesheet" type="text/css" href="Magic8Ball.css">
+    <link href="https://fonts.googleapis.com/css?family=Indie+Flower" rel="stylesheet">
+           <center>
+           		<img src = "https://s3.amazonaws.com/lillypiri/blog/8ball/8ball.png" id="8ball"/>
+           	</center>
+       <p></p>
+   </body>
+</html>
+<script>
+
+   const outputs = [
+       "Reply hazy try again"
```

Comment:
Best practices would be to have the string and comma on the same line. Also, use two space indentation.

```js
{ 
  "Reply hazy try again",
  "Ask again later",
  "Better not tell you now"
}
```